### ISSUE TITLE
It can not load content of a database.

### ISSUE THREAD

**Issue by PhotographerGuo** [fastogt/fastonosql#69]
I have a database nearly 4GB and I can not load the key for every database. When I run DBKCOUNT , it shows 10:55:14.602: DBKCOUNT function error: Invalid argument: merge_operator_ must be set.

**Comment by topilski** [comment]
I think, this issue fixed.

**Comment by PhotographerGuo** [comment]
Hi,

Sorry for delay of reply. 

I build a new and try it, then it shows the same problem: 

17:52:02.453: SCAN function error: Invalid argument: merge_operator_ must be set.


Best Regards,
Fengyang Guo

**Comment by topilski** [comment]
D

**Comment by PhotographerGuo** [comment]
Yes, nearly but I have not tried all merge operator. It quitting immediately when I open. I think I should re install it...

**Comment by PhotographerGuo** [comment]
I have installed again and it keeps quitting...

**Comment by topilski** [comment]
Which version you using? also please cat your ~/fasonosql.log file , i need to see errors

**Comment by PhotographerGuo** [comment]
OX 10.12 . Where is this log file, I have not found..

**Comment by PhotographerGuo** [comment]
1.22.3


Best Regards,
Fengyang Guo

**Comment by topilski** [comment]
Application closed when you trying to connect to RocksDB?

**Comment by PhotographerGuo** [comment]
It can not be opened now. When I click it, it shows one second and quit. 
Best Regards,
Fengyang Guo

**Comment by topilski** [comment]
Ok, remove config file ~/FastoNoSQL/config*

**Comment by PhotographerGuo** [comment]
You mean this : config_new_pro.ini ?
Best Regards,
Fengyang Guo

**Comment by topilski** [comment]
Yes

**Comment by PhotographerGuo** [comment]
It works now. What is the problem?

I have tried merge operator stringappendiota, it shows:
18:42:34.712: SCAN function error: Invalid argument: merge_operator_ must be set.



I have tried 
Best Regards,
Fengyang Guo

**Comment by topilski** [comment]
Can you also cahnge in settings "default" database name to database whith you wanna to work.

**Comment by PhotographerGuo** [comment]
Where is the setting? I have not found “setting”  in software.

Best Regards,
Fengyang Guo

**Comment by topilski** [comment]
Oy, sorry this functionality not implemented for RocksDB and LevelDB

**Comment by topilski** [comment]
Now, implemented, i opened your database on Linux without problems.

**Comment by topilski** [comment]
ping
